Makefile

```diff
@@ -11,7 +11,8 @@ install: get_vendor_deps
 	@go install ./cmd/tendermint
 
 build:
-	go build -o build/tendermint ./cmd/tendermint
+	go build --ldflags '-extldflags "-static"' \
```

Comment:
I dont think so. But it might be nice to have in `make install` for reference when people are installing development version